src/Pim/Component/Catalog/Model/ProductValueCollectionInterface.php

```diff
@@ -0,0 +1,168 @@
+<?php
+
+namespace Pim\Component\Catalog\Model;
+
+/**
+ * Business collection interface to handle product values.
+ * It does not extends \IteratorAggregate as indexation could be performed internally
```

Comment:
So does it extends or not? Here you say "does not", but below you have `extends \Countable, \IteratorAggregate`